### PR TITLE
Add additional documentation on the myths package

### PR DESCRIPTION
**Pull request by willingc**
As @emeeks and I are working on some extensions to nteract, we're learning the myths and mythic- packages. I've added some headings and clarifications to the documentation.

### CODE REVIEW

File: packages/myths/README.md [hunk 1]
```diff
@@ -16,7 +23,11 @@ $ npm install --save myths
 
 ## Usage
 
-First, create a `MythicPackage` with a name, a type for its private state, and the initial state:
+### MythicPackage
+
+First, create a `MythicPackage` with a name, a type for its private state, and the initial state.
+As an example, the following creates a `MythicPackage` named `"iCanAdd"` which uses the `number`
+type for its private state `sum` and an initial state of `sum` as `0:
```
Comment (willingc) [inline packages/myths/README.md]:
```suggestion
type for its private state `sum` and an initial state of `sum` as `0`:
```

File: packages/myths/README.md [hunk 2]
```diff
@@ -2,7 +2,14 @@
 
 **This is a pre-alpha level package; interfaces are not stable yet!**
 
-This framework allows for integrating sets of closely related actions, reducers and epics while mainting high DRY and minimizing dependencies. It provides for a structured way to avoid boilerplate. 
+The `myths` framework allows for integrating sets of closely related actions, reducers and epics. Myths allow
+close relationships where DRY and dependencies are minimized. Therefore, Myths provide for a structured way
+to avoid boilerplate.
+
+As a refresher, Redux helps you maintain application state. In Redux, actions and reducers provide
```
Comment (captainsafia) [inline packages/myths/README.md:11]: Add a preface here that says "Myths build on top of the Redux and RxJS libraries that are used elsewhere in the nteract core SDK."

File: packages/myths/README.md [hunk 3]
```diff
@@ -2,7 +2,14 @@
...
+predictable state management. The state may only be changed by dispatching an action to a reducer.
+In Redux-Observable, an epic is a function that takes in a stream of actions and returns a stream
+of actions.
```
Comment (captainsafia) [inline packages/myths/README.md]: Let's add links to the sites for Redux and RxJS.

File: packages/myths/README.md [hunk 4]
```diff
@@ -30,7 +41,11 @@ export const iCanAdd = createMythicPackage("iCanAdd")<
 });
 ```
 
-You can the use the package to create a `Myth`, again with a name, a type for its payload and optionally a reducer operating on its package's private state:
+### Myth
+
+Next, you can the use the `MythicPackage` to create a `Myth`with a name, a type for its payload, and optionally a reducer
```
Comment (captainsafia) [inline packages/myths/README.md]:
```suggestion
Next, you can the use the `MythicPackage` to create a `Myth` with a name, a type for its payload, and optionally a reducer
```

**Review by captainsafia — APPROVED** [review]
Left some suggestions inline -- looks good otherwise!

**Review by CrystallineCat — APPROVED** [review]
Thank you for improving the README! This looks fine to me.

**Pull request merged**